eidas-saml-parser/src/main/java/uk/gov/ida/notification/eidassaml/EidasSamlApplication.java

```diff
@@ -157,4 +160,13 @@ private String getX509EncryptionCert(EidasSamlParserConfiguration configuration)
 
         return x509EncryptionCert;
     }
+
+    private void registerInjections(Environment environment) {
+        environment.jersey().register(new AbstractBinder() {
```

Comment:
Does Inteliji let you do this as a lambda?

Reply:
No, it displays it as a lambda but if you type in what it displays, it complains it can't find a method.